source/windows/process.c

```diff
@@ -14,8 +14,46 @@
  */
 #include <aws/common/process.h>
 
+#include <aws/common/string.h>
+#include <ctype.h>
 #include <process.h>
+#include <stdio.h>
+#define MAX_BUFFER_SIZE (2048)
 
 int aws_get_pid(void) {
     return _getpid();
 }
+
+static bool s_isspace(uint8_t c) {
```

Comment:
We have isspace(uint8_t) in like 5 places in our code base, we should just add it to common somewhere.